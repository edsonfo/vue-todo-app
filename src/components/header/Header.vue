<template>
  <v-container>
    <v-row class="-centtexter">
      <v-col class="mb-4">
        <h1 class="title-todo">
          todos
        </h1>
      </v-col>
    </v-row>
  </v-container>
</template>

<script>
  export default {
    name: 'Header'
  }
</script>
<style lang="less" scoped>
  .title-todo {
    font-family: sans-serif;
    font-size: 130px;
    font-weight: 100;
    text-align: center;
    color: rgba(175, 47, 47, 0.15);
    text-rendering: optimizeLegibility;
  }
</style>
