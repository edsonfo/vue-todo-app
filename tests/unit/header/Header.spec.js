import { shallowMount } from '@vue/test-utils'
import Header from '@/components/header/Header.vue'

describe('HelloWorld.vue', () => {
  const wrapper = shallowMount(HelloWorld)
  it('should match snapshot', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
