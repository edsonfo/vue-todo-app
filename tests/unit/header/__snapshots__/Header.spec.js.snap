// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component tests should match snapshot 1`] = `
<v-container-stub
  tag="div"
>
  <v-row-stub
    class="-centtexter"
    tag="div"
  >
    <v-col-stub
      class="mb-4"
      tag="div"
    >
      <h1
        class="title-todo"
      >
        
        todos
      
      </h1>
    </v-col-stub>
  </v-row-stub>
</v-container-stub>
`;
